architecture.md

```diff
@@ -331,7 +329,7 @@ cannot be deleted unless the OCI runtime is explicitly being asked to, by using
 
 If the sandbox is not stopped, but the particular container process returned on
 its own already, the `kata-runtime` will first go through most of the steps a `kill`
-would go through for a termination signal. After this (or simply this if the sandboxIDwas already stopped), then `kata-runtime` will: If the sandbox was already stoppedfollowed by:
+would go through for a termination signal. After this (or simply this if the sandboxID was already stopped), then `kata-runtime` will: If the sandbox was already stopped followed by:
```

Comment:
Including "if the sandbox was already stopped" twice in this sentence seems redundant, but I'm concerned I will change the meaning here :) I'll suggest the following rewrite: 
After this process, or if the sandboxID was already stopped to begin with, `kata-runtime` will:

Reply:
@nitkon - this looks like the last block on this PR.

Reply:
@jodh-intel @klynnrif : Updated my PR. Please take a look.